src/main/java/emissary/util/TimeUtil.java

```diff
@@ -232,6 +234,53 @@ public static ZonedDateTime getZonedDateFromISO8601(@Nullable final String dateS
         return dateString == null ? null : ZonedDateTime.parse(dateString, DATE_ISO_8601);
     }
 
+    /**
+     * Parses hex date string into formatted string ("yyyy-MM-dd HH:mm:ss.SSS")
+     *
+     * @param hexDate hex string representation of the date
+     * @return converted time in string format yyyy-MM-dd HH:mm:ss.SSS
+     * @throws DateTimeParseException if could not parse date
+     *
+     */
+    public static String convertHexDate(String hexDate) {
```

Comment:
(Optional, since it's outside the original scope of this PR)
This is a short method, but it contains a lot of magic values.

In the `hexNumber` assignment, would we replace 2` with a constant named `HEX_PREFIX_LEN`? 
In these `parseLong` calls, can we replace `16` with an equivalent constant named `HEX_RADIX`?  (We would _not_ use that constant in the `substring(..)` call.)

The `startingDate` String (or even better, an equivalent `LocalDateTime`) should also be a constant. 

```java
final static int HEX_PREFIX_LEN = "0x".length();`
final static int HEX_RADIX = 16;
final static String STARTING_DATE = "1900-01-01 00:00:00.000";
```

Reply:
It looks like these updates were already made actually. I'll let @drivenflywheel weigh in.